asset/tmp

```diff
@@ -0,0 +1,15 @@
+SHELL=/bin/bash -e -x -o pipefail
```

Comment:
Is this file a draft Makefile that was not meant to be committed?

Reply:
Yes, that's right. I will remove it now